src/components/program/ProgramCourses.jsx

```diff
@@ -50,10 +52,16 @@ const ProgramCourses = () => {
                 </Collapsible.Trigger>
 
                 <Collapsible.Body className="collapsible-body mt-3 ml-4.5">
-                  <div className="course-card-result mb-2">
-                    <FontAwesomeIcon icon={faCalendarAlt} className="fa-calendar-alt mr-2" />
-                    {courseRun && <span className="font-weight-bold">Starts {moment(courseRun.start).format(DATE_FORMAT)}</span>}
-                  </div>
+                  {
+                    (courseRun?.pacingType === PROGRAM_PACING_MAP.INSTRUCTOR_PACED && courseRun.start)
+                    && (
+                      <div className="course-card-result mb-2">
+                        <FontAwesomeIcon icon={faCalendarAlt} className="fa-calendar-alt mr-2" />
+                        {courseRun && <span className="font-weight-bold">Starts {moment(courseRun.start).format(DATE_FORMAT)}</span>}
```

Comment:
We can remove the first check `courseRun &&` from here since it's already being checked in the above condition

Reply:
yeah sorry, it was in my mind to remove then slipped somehow. Fixed now